src/controllers/pagoPAController.ts

```diff
@@ -41,10 +41,13 @@ export default class PagoPAController {
     }
 
     const profile = errorOrProfile.value;
-    const maybeCustomEmail = ProfileWithEmail.is(profile)
-      ? profile.email
-      : undefined;
-    const email = maybeCustomEmail ? maybeCustomEmail : profile.spid_email;
+    const maybeCustomEmail =
```

Comment:
I don't know about this, I actually would like to have the app deal with this duality - let's give to the app the data as is (`spid.email` and `cd.email` as separate attributes)